examples/transpiler/BUILD.bazel

```diff
@@ -93,6 +101,27 @@ build_test(
     ],
 )
 
+# Runs babel to transpile ts -> js while tsc would fail with typecheck errors
+# Tagging as manual should prevent the typecheck from running even when requesting
+# transpiler output files.
+ts_project(
+    name = "typecheck_fail",
+    srcs = ["typecheck_fail.ts"],
+    declaration = True,
+    source_map = True,
+    tags = ["manual"],
```

Comment:
This test is extra, I just want to make sure failing tsc is not invoked when tagged as manual, even when the non-tsc transpiler is forced to be invoked.